### PR TITLE
Nano gene

### PR DESCRIPTION
**Pull request by MayhemSailor**
Adds NanoGene, genetics wall dispenser to Yogstation (second try, now that I know how to update my fork, or at least I think I do, remake of  #6684.)

### Introducing NanoGene - The Genetics Wall Dispenser

Nanotrasen Department of Vending has introduced it's newest innovation - NanoGene wall dispensing system. This new design provides geneticists with easy access to the tools and medicines of the trade. We are particularly proud of the addition of Monkey Cubes(TM) to the list of supplies!

We have implemented the installation of NanoGene machines to base NT Stations.

The following products are available:
- Mannitol Pill Bottle: 2
- Mutadone Pill Bottle: 2
- Charcoal Pill Bottle: 2
- Prescription Glasses Box: 1
- Data Disk Box: 1
- Monkey Cubes: 10

Contraband:
- Gorilla Cube: 1
- Radium Bottle: 1
- Mutagen Bottle: 2
- ~~Syndie Monkey Cubes~~

The NanoGene:
wall_monkey wall_monkey1
 wall_monkey_deny wall_monkey_deny1

Genetics Alterations:
Genetics

#### Changelog

:cl:  
rscadd: Introducing the NanoGene - The Genetics Wall Dispenser  
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
Are you sure contraband content like this is good? It's like putting stunprod into engi vendor, but it's also worse because it's based on griefing everyone near until monkeys are dead

**Comment by MayhemSailor** [comment]
> 
> 
> Are you sure contraband content like this is good? It's like putting stunprod into engi vendor, but it's also worse because it's based on griefing everyone near until monkeys are dead

If it becomes an issue it's an easy fix.

**Comment by ghost** [comment]
Unstable mutagen bottles should be part of the contraband.

**Comment by ghost** [comment]
contraband items that are actually traitor gear wow

**Comment by ynot01** [comment]
i have a screwdriver and a multitool so now i get traitor items

**Comment by MayhemSailor** [comment]
Alright, I've adjusted the contraband items.

File: code/modules/vending/monkey_wall.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	products = list(/obj/item/storage/pill_bottle/mannitol = 2,
+		            /obj/item/storage/pill_bottle/mutadone = 2,
+		            /obj/item/reagent_containers/pill/charcoal = 6,
+					/obj/item/clothing/glasses/regular = 5,
+					/obj/item/disk/data = 2,
+					/obj/item/reagent_containers/food/snacks/monkeycube = 10)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/radium = 1,
+					  /obj/item/reagent_containers/glass/bottle/mutagen = 2,
+	                  /obj/item/reagent_containers/food/snacks/monkeycube/gorilla = 1)
+	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
```
Comment (alexkar598) [inline code/modules/vending/monkey_wall.dm:16]: That is some high armour values. Why are they so high?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/vending/monkey_wall.dm (hunk 1 above)
Comment (MayhemSailor) [reply code/modules/vending/monkey_wall.dm:16]: Copypasta of the wall med.

**Pull request merged**